### DISCUSSION TITLE
Change the Zoom In/Out configuration?

### DISCUSSION THREAD

**Discussion by xpresseasy** [highfestiva/finplot#361] [Q&A]
I am using the Trackball mouse, it doesn't have the physical scroll wheel per se. The mouse manufacturer has the emulation software for the scroll wheel but the emulation doesn't seem to work for the finplot plot (for whatever reasons). I am wondering if there is a configuration mapping function for the finplot zoom in and out feature, so these features can be mapped to other keys.

BTW, I can't seem to find the user docs about the operations such as drawing lines or circles or other functions. Please elaborate. Thanks.

**Comment by highfestiva** [comment]
finplot uses the default Qt mouse events, which are handled by the `GraphicsView.wheelEvent()` function. I'm wrapping that to make sure it works in embedded views here: https://github.com/highfestiva/finplot/blob/master/finplot/__init__.py#L2478. If you're able to get some callback when you zoom on our mouse wheel, you can simply call that function.

See `examples/lines.py`.
